GitExternal.cmake

```diff
@@ -113,7 +113,12 @@ function(GIT_EXTERNAL DIR REPO tag)
     if(nok)
       message(FATAL_ERROR "git checkout ${TAG} in ${DIR} failed: ${error}\n")
     else()
-      execute_process(COMMAND "${GIT_EXECUTABLE}" submodule update --init --recursive
+      # init submodules of the subproject, excluding CMake/common if present
+      execute_process(COMMAND "${GIT_EXECUTABLE}" submodule init
+        WORKING_DIRECTORY "${DIR}" OUTPUT_QUIET)
+      execute_process(COMMAND "${GIT_EXECUTABLE}" submodule deinit CMake/common
+        WORKING_DIRECTORY "${DIR}" OUTPUT_QUIET ERROR_QUIET)
```

Comment:
This deinit only applies to subprojects. In for some reason you want to have a clone of CMake/common in a subproject, then yes, you need to init it there. That seems logical to me, do you have a use case in mind where this isn't what you expect?